### PR TITLE
Fix duplicate imports in docs

### LINKED ISSUE #142: Global hotkeys aren't properly grouped in the Hotkeys dialog
### Bug Report

- __Package version(s)__: v1.0.1
- __Browser and OS versions__:
  - Chrome 54.0.2840.71 (64-bit) on OS X 10.11.6
  - Safari 10.0.1 on OS X 10.11.6
  - Firefox 42.0 on OS X 10.11.6

#### Describe the issue:

On the Hotkeys page of the Blueprint docs (http://blueprintjs.com/docs/#components.hotkeys), two overlayed hotkey dialogs when you press `Shift + /` (i.e. `?`)

#### Actual behavior:

image

#### Expected behavior:

The "Focus the Piano" hotkey should be rendered below in the other global hotkeys in the underlying dialog.

### PR DESCRIPTION
**Pull request by cmslewis**
#### PR checklist

- [x] Addresses an existing issue: fixes #142

#### What changes did you make?

Example files in `packages/core` were using relative paths to import files from the core package's `src/` directory, but other files were importing dependencies from `@blueprintjs/core/src` via `npm`. This caused two instances of several Blueprint files to be loaded into the docs page, resulting in funky side effects like the one in #142 (two improperly grouped hotkeys dialogs).

The fix was to import dependencies from `@blueprintjs/core/src`, not `../../src`, in all of the examples files.

#### Is there anything you'd like reviewers to focus on?

- Make sure the paths look okay and that the new scheme makes sense intuitively.
- Also @themadcreator - is it expected behavior for the global hotkeys from different hotkeys targets to show up in an arbitrary order in the dialog? Note that the global-hotkey order is wonky for the piano example in the **After** examples. (EDIT: I just checked this in a pre-open-source version of the docs. The order of global hotkeys then was the same as it is now, so we're good.)

#### Screenshots

**Before:**
screen shot 2016-11-18 at 10 10 09 am
screen shot 2016-11-18 at 10 12 27 am

**After:**
screen shot 2016-11-18 at 10 11 19 am
screen shot 2016-11-18 at 10 11 30 am

### CODE REVIEW

**Comment by adidahiya** [comment]
> The fix was to import dependencies from @blueprintjs/core/src, not ../../src, in all of the examples files.

maybe we should write a custom lint rule for this?

**Comment by cmslewis** [comment]
@adidahiya How do I do that?

**Comment by adidahiya** [comment]
@cmslewis https://palantir.github.io/tslint/develop/custom-rules/ (but not part of this PR)

File: packages/core/package.json [hunk 1]
```diff
@@ -18,6 +18,7 @@
     "react-dom": "^15.0.1 || ^0.14"
   },
   "devDependencies": {
+    "@blueprintjs/core": "^1.0.1",
```
Comment (giladgray) [inline packages/core/package.json]: whoa this is really bad... instead we should set up local resolution in webpack.

**Pull request merged**